### PR TITLE
change(reporting): Set default log level to Info and cleanup reporters

### PR DESCRIPTION
**Pull request by lkts**
Fixes https://github.com/stryker-mutator/stryker-net/issues/159

### CODE REVIEW

File: src/Stryker.CLI/Stryker.CLI/CLIOptions.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static class CLIOptions
             ArgumentName = "--log-console",
             ArgumentShortName = "-l <logLevel>",
             ArgumentDescription = "Sets the logging level | Options [error, warning (default), info, debug, trace]",
```
Comment (richardwerkman) [inline src/Stryker.CLI/Stryker.CLI/CLIOptions.cs]: info should say (default) now

File: src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs [hunk 2]
```diff
@@ -41,12 +41,10 @@ public class InitialisationProcess : IInitialisationProcess
 
         public MutationTestInput Initialize(StrykerOptions options)
         {
-            _reporter.OnInitialisationStarted();
```
Comment (richardwerkman) [inline src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs]: I don't think the _reporter property is needed at all anymore here

**Comment by richardwerkman** [comment]
Thank you for your pull request! I have few remarks.

File: src/Stryker.CLI/Stryker.CLI/CLIOptions.cs (hunk 1 above)
Comment (lkts) [reply src/Stryker.CLI/Stryker.CLI/CLIOptions.cs]: indeed!

File: src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs (hunk 2 above)
Comment (lkts) [reply src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs]: right

**Pull request merged**